SoftLayer/CLI/firewall/list.py

```diff
@@ -70,7 +70,27 @@ def cli(env):
                              'hardwareId')
             ])
 
+    table_gatewalls = formatting.Table(['firewall id',
+                                        'firewall',
+                                        'type',
+                                        'Hostname',
+                                        'Location',
+                                        'Public Ip',
+                                        'Private Ip',
+                                        'Associated vlan',
+                                        'status'], title='Multi Vlan Firewall')
+    fw_gatewwalls = mgr.get_firewalls_gatewalls()
+
+    for gatewalls in fw_gatewwalls:
+        table_gatewalls.add_row([gatewalls.get('id'), gatewalls.get('name'),
```

Comment:
Display the ID with the format `multiVlan:FIREWALL_ID`, where `multiVlan` is constant and `FIREWALL_ID` is the firewall id